EthOn_glossary.md

```diff
@@ -273,6 +276,9 @@ Relates a Post Transaction State to the following Post Transaction State.
 Relates a State to the following State. In EthOn the state transition system has no branches.
 ## O
 ## P
+### Prefunded Account
+[`ethon:PrefundedAccount`](http://ethon.consensys.net/PrefundedAccount)   
+A Prefunded Account is an Account whose initial balance is defined in the protocol of the Blockchain. In the initial Ethereum Blockchain those that participated in the crowd funding received Prefunded Accounts. However, Prefunded Accounts could be added with any hard fork.
```

Comment:
Hi axic, that is a good point, I'll also add a property for preset code. Many EIPs suggest that this is coming.
This also implies the term `prefunded` doesn't make sense anymore. I don't much like `Irregularly` either - to me anything that the protocol specifies is regular.
I'll have to think about it. If you have other suggestions, please come forward.
Thanks!